src/common/options.cc

```diff
@@ -955,9 +955,14 @@ std::vector<Option> get_global_options() {
     .set_default(512)
     .set_description("Size of queue of incoming connections for accept(2)"),
 
-    Option("ms_tcp_read_timeout", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+
+    Option("ms_connection_ready_timeout", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
+    .set_default(600)
```

Comment:
Ten minutes seems like a long time to be attempting to open a connection... it seems like we'd want to fault and retry/reconnect after maybe 10s of seconds at most?  Or am I misunderstanding the situations where this timeout comes up?

Reply:
@liewegas 10s is fine with me:-)  10 minutes might be over-conservative...